src/applications/simple-forms/mock-simple-forms-patterns/config/form.js

```diff
@@ -36,20 +36,22 @@ import {
 } from '../pages/mockArrayMultiPageBuilderPages';
 import { MockCustomPage, mockCustomPage } from '../pages/mockCustomPage';
 import arrayBuilderPatternChooseFlow from '../pages/mockArrayMultiPageBuilderChooseFlow';
+import NewConfirmationPage from '../../form-upload/containers/ConfirmationPage.new';
```

Comment:
Should we have a canonical v2 confirmation page live somewhere else and both the mock pattern and form upload apps refer to that?

Reply:
Eventually we will but this is still very much a work in progress. The whole point of adding this is for testing. Once the confirmation page is fully complete, we'll be replacing the existing confirmation page with it.